tools/importer-rest-api-specs/components/parser/models.go

```diff
@@ -656,9 +656,6 @@ func (d SwaggerDefinition) parseNativeType(input *spec.Schema) *models.ObjectDef
 		}
 	}
 
-	// TODO: Objects with Format of File are actually RawFiles..
-	// "type": "object",
-	// "format": "file"
 	if input.Type.Contains("object") {
 		return &models.ObjectDefinition{
```

Comment:
thinking about it, it'd make more sense to move this logic here, leaving the change above as-is and making this:

```suggestion
		if strings.EqualsFold(input.Format, "file") {
			return &models.ObjectDefinition{
				Type: models.ObjectDefinitionRawFile,
			}
		}
		return &models.ObjectDefinition{
```